controllers/account/account_controller.go

```diff
@@ -36,7 +35,8 @@ import (
 )
 
 var log = logf.Log.WithName("controller_account")
-var AssumeRole = AssumeRoleFunc
+var AssumeRoleAndCreateClient = stsclient.AssumeRoleAndCreateClient
+var handleRoleAssumption = stsclient.HandleRoleAssumption
```

Comment:
I understand making `AssumeRuleAndCreateClient` a public var so it can be overwritten in tests, but why assign to a private variable for `handleRoleAssumption` - it doesn't seem to require a global variable as it only gets used inside this file?

Reply:
For OSD-14200 it will use the `stsclient.HandleRoleAssumption` function in the account_validation_controller